js/modules/circles.js

```diff
@@ -43,10 +53,37 @@ function create_circles(dv_stats_data, canvas_context, scale_colors_fxns, fig_cf
       }
       
     }
+    
+    // skip the rest of this loop and go to the next one
+    // the selected points will be drawn after so they are on top
+    if(color_category === selected_color) {
+      continue;
+    }
+    
     // finish the current path (this is what actually does the final drawing)
     canvas_context.fill();
     canvas_context.stroke();
   }
+  
+  // Now draw any selected_color_points so that they are on top
+  if(selected_color_points) {
+    canvas_context.beginPath();
+    for (let i in selected_color_points) {
+      var selected_point = selected_color_points[i],
+          selected_color_point = scale_colors_fxns.circles(selected_point.per);
+      canvas_context.fillStyle = selected_color_point;
+      canvas_context.strokeStyle = selected_color_point;
+      radius = 4; // radius of the selected points is bigger
```

Comment:
do you want to pull this larger radius out as a variable defined at the top of this file?

Reply:
Do you mean have two different ones - `radius_reg` and `radius_selected`? I can do that 👍